templates/common/infra/bicep/core/ai/hub-dependencies.bicep

```diff
@@ -0,0 +1,170 @@
+param location string = resourceGroup().location
+param tags object = {}
+
+@description('Name of the key vault')
+param keyVaultName string
+@description('Name of the storage account')
+param storageAccountName string
+@description('Name of the OpenAI cognitive services')
+param openAiName string
```

Comment:
I believe we are using openAI instead of openAi in other templates.  Can you check?

Reply:
We don't have any `openAI` references in `azure-dev` right now.

Reply:
Looks like we are using `openAi` in `contoso-chat` so will stick with that for now.